packages/template-retail-react-app/app/routes.jsx

```diff
@@ -89,7 +90,7 @@ export const routes = [
         exact: true
     },
     {
-        path: '/social-callback',
+        path: config.login.social?.redirectURI || '/social-callback',
```

Comment:
If the user sets the `redirectURI` to be a different relative path in their config file, we want this reflected here in `routes.jsx`. The default will be `/social-callback`